Mapify/Patches/MainMenuPatch.cs

```diff
@@ -0,0 +1,14 @@
+﻿using DV.UI;
+using HarmonyLib;
+
+namespace Mapify.Patches
+{
+    [HarmonyPatch(typeof(MainMenu), nameof(MainMenu.GoBackToMainMenu))]
+    public static class MainMenuPatch
+    {
+        private static void Postfix()
+        {
+            StreamedObjectInitPatch.ResetStreamers();
```

Comment:
This should be hooked into our existing map cleanup logic

Reply:
done